mkauthlist/mkauthlist.py

```diff
@@ -497,6 +529,53 @@ def write_contributions(filename,data):
             authors.append(author)
         params = dict(defaults,authors=''.join(authors))
 
+    # AASTEX 7 updates
+    if cls in ['aastex7']:
+        document = aastex7_document
+        authlist = aastex7_authlist
+        # Assume first author is the corresponding author
+        corrauthor = None
+        author_email = {}
+
+        for i,d in enumerate(data):
+            if d['Affiliation'] == '':
+                logging.warn("Blank affiliation for '%s'"%d['Authorname'])
+            if d['Authorname'] == '':
+                logging.warn("Blank authorname for '%s %s'"%(d['Firstname'],
+                                                             d['Lastname']))
+
+            if corrauthor is None:
+                corrauthor = d['Authorname']
+            authorkey = '{%s}'%(d['Authorname'])
+
+            if args.orcid and d['ORCID']:
+                authorkey = "[%s, gname='%s', sname='%s']"%(d['ORCID'],d['Firstname'],d['Lastname']) + authorkey
+            else:
+                # Appears to be a bug in the AASTeX such that omitting one of them kills the compilation.
+                authorkey = "[0000-0000-0000-0000,gname='%s', sname='%s']"%(d['Firstname'],d['Lastname']) + authorkey
+
+            if authorkey not in authdict.keys():
+                authdict[authorkey] = [d['Affiliation']]
+            else:
+                authdict[authorkey].append(d['Affiliation'])
+
+            author_email[authorkey] = d['Email']
+            #if d['Authorname'] not in authdict.keys():
+            #    authdict[d['Authorname']] = [d['Affiliation']]
+            #else:
+            #    authdict[d['Authorname']].append(d['Affiliation'])
+
```

Comment:
Can probably remove this commented code?

Reply:
correct on both counts, I'll delete both.